src/EnergyPlus/DXCoils.hh

```diff
@@ -384,11 +384,11 @@ namespace DXCoils {
         Array1D<Real64> MSLatentCapacityTimeConstant; // Time constant for latent capacity to reach steady state
         Array1D<Real64> MSFanPowerPerEvapAirFlowRate;
         Array1D<Real64> MSFanPowerPerEvapAirFlowRate_2023;
-        Real64 FuelUsed;         // Energy used, in addition to electricity [W]
-        Real64 FuelConsumed;     // Energy consumed, in addition to electricity [J]
-        Real64 MSFuelWasteHeat;  // Total waste heat [J]
-        bool MSHPHeatRecActive;  // True when entered Heat Rec Vol Flow Rate > 0
-        int MSHPDesignSpecIndex; // index to MSHPDesignSpecification object used for variable speed coils
+        Real64 FuelUsed = 0.0;     // Energy used, in addition to electricity [W]
+        Real64 FuelConsumed = 0.0; // Energy consumed, in addition to electricity [J]
```

Comment:
`FuelConsumed` was uninitialized and is used in the [heat emissions](https://github.com/NREL/EnergyPlus/blob/16d5a46c06fca90ed28a38e654748b1149b31d88/src/EnergyPlus/OutputReportTabular.cc#L4375-L4379) calculation for Coil:Heating:DX:MultiSpeed. Why other files with the same coil don't trip the same table diffs, I have no idea.

Reply:
I wonder if there is some other initialization of the variable for those other coil types, but not in this particular code path.  Either way this seems just fine.